docs/relay/relay.md

```diff
@@ -7,42 +7,42 @@ Netcode for GameObjects (Netcode) allows you to connect to a host by its IP and
 
 Many factors impact how you connect to the remote host, and it can be tricky to implement. There are two ways of achieving such a connection: 
 - [NAT punching](../learn/listenserverhostarchitecture.md#option-c-nat-punchthrough): an advanced technique that allows you to directly connect to the host computer even if it is on another network.  
-- Usage of a Relay server: the server is on the Internet with a public-facing IP that you and the host can reach. After the connection to the Relay, it will pass along the messages to all the other party members connected.
+- Usage of a Relay server: the server is on the Internet with a public-facing IP that you and the host can reach. After each side binds with the Relay, they can establish a connection and send data to each other via the Relay.
 
 Netcode does not offer tools to help you do a successful punch through a NAT. But, Unity Services provides a Relay Service that can relay all Unity Transport based technology, like Netcode.
 
 ## How does Relay work?
 
 You need to install the Unity Relay SDK if you want to interact with the service. To install it, add `com.unity.services.relay` package to your project.
 
-The host of your game needs to allocate a Relay first. This process will create the Relay server and allow the clients to connect to it. Once done, you will be able to get a join code from your server. It is a random string that your clients will provide to the SDK to join the correct server.
+The host of your game needs to request a Relay allocation first. This process will reserve capacity on a Relay server that will allow the clients to bind to it. Once done, you will be able to get a join code for the allocation. It is a random string that your clients will provide to the SDK to join the correct server and allocation.
```

Comment:
perhaps we can use "prepare capacity" instead of "reserve capacity", just in case we overcommit at some point? Or we can update then, I don't think this document would be legally binding?